### DISCUSSION TITLE
I suggested Excalibur.js on EnginesDatabase❗

### DISCUSSION THREAD

**Discussion by tenpaMk2** [excaliburjs/Excalibur#2789] [Show and tell]
Today, I found [EnginesDatabase](https://enginesdatabase.com/) which is database web site of game dev engines.

I found Phaser.js and MelonJS, but not Excalibur.js.
So, I suggested it from "Suggest Engine" buttons at top of this site.

Maybe, few days later, Excalibur.js is shown in this site. Stay tuned❗

![Screenshot of EnginesDatabase](https://github.com/excaliburjs/Excalibur/assets/8945311/d27ed672-1fd6-41fb-8963-7a20110e390f)

**Comment by eonarheim** [comment]
@tenpaMk2 Many thanks! I'll keep an eye out!

**Reply by eonarheim** [reply]
Looks like it's there! 
<img width="241" alt="image" src="https://github.com/excaliburjs/Excalibur/assets/612071/288fec7a-8a3a-400d-b6e8-e406e590d646">
